### ISSUE TITLE
Windows 11 ERROR: IOError: chmod("mySite\\.gitignore", 0o644): invalid argument (EINVAL)

### ISSUE THREAD

**Issue by dstorti** [tlienart/FranklinTemplates.jl#157]

**Comment by tlienart** [comment]
PS: are you somehow on a non-conventional drive? (the EINVAL thing is a bit odd)

**Comment by dstorti** [comment]
Yes. It was on  Google cloud drive.  I will try from my own hard drive on Monday. Thanks!
--Duane

On July 29, 2022 10:01:51 AM PDT, Thibaut Lienart ***@***.***> wrote:
>PS: are you somehow on a non-conventional drive? (the EINVAL thing is a bit odd)
>
>-- 
>Reply to this email directly or view it on GitHub:
>https://github.com/tlienart/FranklinTemplates.jl/issues/157#issuecomment-1199756611
>You are receiving this because you authored the thread.
>
>Message ID: ***@***.***>
-- 
Sent from my Android device with K-9 Mail. Please excuse my brevity.
